### PR TITLE
Remove Omicron story

### LINKED ISSUE #722: Remove Omicron story
The [Omicron story](https://cov-spectrum.org/stories/omicron) page can be removed. It's not useful anymore (and not sure if it has ever been).

### PR DESCRIPTION
**Pull request by JonasKellerer**
Resolves: #722

### CODE REVIEW

**Comment by fengelniederhammer** [comment]
The `VariantInternationalComparisonMap` is now unused (including the world map). @chaoran-chen Should we delete it? Or is there any chance that it will be used somewhere else in the future?

**Comment by chaoran-chen** [comment]
Eventually, it would be nice to add more maps including a world map to the "normal" variant pages but I don't think that we will do it very soon, so I think that we can remove the component.

File: src/maps/RegionMap.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
```
Comment (fengelniederhammer) [inline src/maps/RegionMap.tsx]: I think we can delete the whole file, can't we? Including the `world.json` next to it, as well as the dependency on `react-simple-maps`?

Comment (JonasKellerer) [reply src/maps/RegionMap.tsx]: Yes. It is no longer needed in other files.

**Comment by fengelniederhammer** [comment]
Looks good to me :+1:

**Review by chaoran-chen — APPROVED** [review]
👍

**Pull request merged**